doc/modules/model_evaluation.rst

```diff
@@ -163,6 +159,24 @@ the :func:`fbeta_score` function::
     >>> grid = GridSearchCV(LinearSVC(dual="auto"), param_grid={'C': [1, 10]},
     ...                     scoring=ftwo_scorer, cv=5)
 
+==============================================     ==================================
+Function                                           Comment
+==============================================     ==================================
+**Classification**
+:func:`metrics.fbeta_score`                        implements parameter ``beta``
+:func:`metrics.hamming_loss`
+:func:`metrics.zero_one_loss`
+
+**Regression**      
+:func:`metrics.root_mean_squared_log_error`
+:func:`metrics.mean_poisson_deviance`
+:func:`metrics.mean_gamma_deviance`
+:func:`metrics.mean_tweedie_deviance`              implements parameter ``power``
+:func:`metrics.mean_pinball_loss`                  implements parameter ``alpha``
+:func:`metrics.d2_tweedie_score`                   implements parameter ``power``
+:func:`metrics.d2_pinball_score`                   implements parameter ``alpha``
+:func:`metrics.d2_absolute_error_score`            
+==============================================     ==================================
```

Comment:
I do see your point and I have gone back and forth on this in my thinking.

However, out of four metrics that remain in the table after your suggestions (`mean_tweedie_deviance`,              `mean_pinball_loss`, `d2_tweedie_score`,  `d2_pinball_score`), none actually _require_ the user to set a parameter value, all have default values implemented. E.g. this doesn't throw an error when passed to `scoring`:

```pinball_scorer = metrics.make_scorer(metrics.d2_pinball_score)```

The only one that doesn't behave this way is the `fbeta_score` among the classification scores where parameter `beta` does not have the default value.

I am happy to accept your suggestions but this inconsistency did confuse me some.

Reply:
I think it's also fine to keep the behavior of `metrics.make_scorer(metrics.d2_pinball_score)` unchanged for the sake of backward compatibility, even if I would not recommend users to follow this pattern in our documentation because it makes hard/implicit to find out what value for `alpha` is used in this case.

Reply:
Thank you for clarifying, makes sense. The last commit reflects your initial suggestions, I think it should all be consistent now?